npm/lit/src/mount.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  injectStylesBeforeElement,
+  StyleOptions,
+  getContainerEl,
+  setupHooks,
+} from '@cypress/mount-utils'
+import { LitElement, render, RootPart, TemplateResult } from 'lit'
+
+const DEFAULT_COMP_NAME = 'unknown'
+
+type LitElementConstructor<T> = new (...args: any[]) => T;
+
+export interface MountOptions extends
+  Partial<StyleOptions> {
+  log?: boolean
+}
+
+let componentInstance: LitElement | undefined
+
+const cleanup = () => {
+  componentInstance?.remove()
+}
+
+// Extract the component name from the object passed to mount
+const getComponentDisplayName = <T extends LitElement>(Component: LitElementConstructor<T>): string => {
+  if (Component.name) {
+    const [_, match] = /Proxy\<(\w+)\>/.exec(Component.name) || []
+
+    return match || Component.name
+  }
+
+  return DEFAULT_COMP_NAME
+}
+
+/**
+ * Mounts a Lit element inside the Cypress browser
+ *
+ * @param {LitElementConstructor<T>} Element Lit element being mounted
+ * @param {MountReturn<T extends SvelteComponent>} options options to customize the element being mounted
+ * @returns Cypress.Chainable<MountReturn>
+ *
+ * @example
+ * import Counter from './Counter'
+ * import { html } from 'lit'
+ * import { mount } from 'cypress/svelte'
+ *
+ * it('should render', () => {
+ *   mount(Counter, html`<my-counter .count=${32}></my-counter>`)
+ *   cy.get('button').contains(42)
+ * })
+ */
+export function mount<T extends LitElement> (
+  Component: LitElementConstructor<T>,
+  Template: TemplateResult,
```

Comment:
If the user registers their custom element with a side effect import, it should not be necessary to even pass in the component class into `mount()`.

I think taking in a `TemplateResult` is not a bad start. It does mean user has to import `html` from `lit` in their test code. (Which I'd personally prefer anyway since it'll come with some checks with lit-plugin). But the API could also support the user providing just a string, then one can wrap it with lit's `unsafeHTML` directive and provide to `html` tag function.

This is the same take as open-wc's test fixture approach.